### PR TITLE
Add Kokoro config and build files

### PR DESCRIPTION
**Pull request by dmytro-chernov**
1. Add Kokoro continuous and pre-submit config files
2. Add Kokoro build scripts

### CODE REVIEW

File: kokoro/gcp_ubuntu/continuous.cfg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# -*- protobuffer -*-
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/continuous.cfg:1]: Please add license headers to all source code files.

File: kokoro/gcp_ubuntu/kokoro_build.sh [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+termInstance() {
+  gcloud compute instances delete teradata-kokoro --zone us-central1-a --project="${GCP_PROJECT}" --quiet
+}
+
+#delete instance after error
+trap 'termInstance' ERR
+
+ls "${KOKORO_KEYSTORE_DIR}"
+#Variables
+export TD_PSW=$(<"${KOKORO_KEYSTORE_DIR}"/76474_teradata-12232021)
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/kokoro_build.sh:19]: Please enclose dynamic arguments in quotes to avoid interpretation issues due to unexpected spaces (I.e., `TD_PSW="$(...)"` in this case.

File: kokoro/gcp_ubuntu/kokoro_build.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+# WARNING: please only enable 'set -x' if necessary for debugging, and be very
+#  careful if you handle credentials (e.g. from Keystore) with 'set -x':
+#  statements like "export VAR=$(cat /tmp/keystore/credentials)" will result in
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+
+#Create Kokoro Teradata instance
+gcloud compute instances create "${TD_HOST}" --project="${GCP_PROJECT}" --zone=us-central1-a --machine-type=n2-standard-2 \
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/kokoro_build.sh:47]: The line is a bit long. I think one line per argument pair would make this more readable.

File: kokoro/gcp_ubuntu/kokoro_build.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+sleep 5m
+
+#Bugfix for bazel 
+use_bazel.sh 4.1.0
+command -v bazel
+bazel version
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/dwh-assessment-extraction-tool"
+
+rm -r /home/kbuilder/.cache/bazel/_bazel_kbuilder/install/4cfcf40fe067e89c8f5c38e156f8d8ca
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/kokoro_build.sh:62]: That seems very specific. Why is this removed? Will this break if there is a new version of whatever this is?

File: kokoro/gcp_ubuntu/kokoro_build.sh [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+cd "${KOKORO_ARTIFACTS_DIR}/github/dwh-assessment-extraction-tool/bazel-bin/dist/dwh-assessment-extraction-tool"
+
+mkdir output
+
+#Extract data from TD instance
+set +x
+./dwh-assessment-extraction-tool td-extract --db-address jdbc:teradata://"${TD_HOST}"/DBS_PORT=1025,DATABASE="${TD_DB}" --output "${EXPORT_PATH}"  --db-user "${TD_USR}" --db-password "${TD_PSW}"
+
+
+#cd "${KOKORO_ARTIFACTS_DIR}/github/dwh-assessment-extraction-tool/integ-tests/"
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/kokoro_build.sh:79]: Can this be removed?

File: kokoro/gcp_ubuntu/kokoro_build_presubmit.sh [hunk 6]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/kokoro_build_presubmit.sh:1]: There seems to be a bit of duplication going on. Would it make sense to move some of it to a common library?

File: kokoro/gcp_ubuntu/kokoro_build.sh [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+# set -x
+
+#Create Kokoro Teradata instance
+gcloud compute instances create "${TD_HOST}" --project="${GCP_PROJECT}" --zone=us-central1-a --machine-type=n2-standard-2 \
+--network-interface=subnet=default,no-address --maintenance-policy=MIGRATE --service-account="${GCP_SERVICE_ACCOUNT}" \
+--scopes="${GCP_SCOPES}" --create-disk=auto-delete=yes,boot=yes,device-name="${TD_HOST}",image="${GCP_IMAGE}",mode=rw,size=300,type=projects/"${GCP_PROJECT}"/zones/us-central1-a/diskTypes/pd-balanced \
+--reservation-affinity=any
+
+#Time to allow TD to recover
+sleep 5m
```
Comment (wfreliszka) [inline kokoro/gcp_ubuntu/kokoro_build.sh:53]: Is creating instance async? If so how do we know if this succeeded? 

Can min wait be replaced with dynamic check? Also, how long does the downloading and extracting take?

File: kokoro/gcp_ubuntu/continuous.cfg (hunk 1 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/continuous.cfg:15]: Added the license headers

File: kokoro/gcp_ubuntu/kokoro_build.sh (hunk 2 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build.sh]: Wrapped in double quotes

File: kokoro/gcp_ubuntu/kokoro_build.sh (hunk 5 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build.sh]: Removed.

File: kokoro/gcp_ubuntu/kokoro_build_presubmit.sh (hunk 6 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build_presubmit.sh]: Was able to get the Kokoro job name from the sys. variable and create a unified script.

File: kokoro/gcp_ubuntu/kokoro_build.sh (hunk 7 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build.sh]: By default, that command runs in sync mode and waits for the results.
I've tried to verify, but there are some problems with remote ssh connection and host validation.

File: kokoro/gcp_ubuntu/kokoro_build.sh (hunk 4 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build.sh]: Removed. Fixed by using the specific bazel version, not the default one.
use_bazel.sh 4.1.0

File: kokoro/gcp_ubuntu/kokoro_build.sh (hunk 3 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build.sh]: Updated the formatting.

File: kokoro/gcp_ubuntu/kokoro_build.sh [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+export GCP_IMAGE="projects/${GCP_PROJECT}/global/images/teradata1610-ubuntu20"
+export GCP_SCOPES="https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,"\
+"https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/servicecontrol,"\
+"https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append"
+
+gcloud components update
+
+#Create Kokoro Teradata instance
+gcloud compute instances create "${TD_HOST}" --project="${GCP_PROJECT}" \
+--zone=us-central1-a \
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/kokoro_build.sh:62]: I would indent this arguments.

File: kokoro/gcp_ubuntu/kokoro_build.sh [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+sleep 4m
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/dwh-assessment-extraction-tool/bazel-bin/dist/dwh-assessment-extraction-tool"
+
+mkdir output
+
+
+#Extract data from TD instance
+set +x
+./dwh-assessment-extraction-tool td-extract --db-address jdbc:teradata://"${TD_HOST}"/DBS_PORT=1025,DATABASE="${TD_DB}" --output "${EXPORT_PATH}"  --db-user "${TD_USR}" --db-password "${TD_PSW}"
```
Comment (haselbach) [inline kokoro/gcp_ubuntu/kokoro_build.sh:97]: I'd break up this line using one line per argument pair.

**Review by haselbach — APPROVED** [review]

File: kokoro/gcp_ubuntu/kokoro_build.sh (hunk 9 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build.sh]: Updated. Thank you.

File: kokoro/gcp_ubuntu/kokoro_build.sh (hunk 8 above)
Comment (dmytro-chernov) [reply kokoro/gcp_ubuntu/kokoro_build.sh]: Updated

**Review by haselbach — APPROVED** [review]
